src/main/kotlin/org/jetbrains/bio/big/Bed.kt

```diff
@@ -53,8 +57,16 @@ data class BedEntry(
             .result()
 
     /**
-     * Parses minimal representation
-     * @param fieldsNumber Expected BED format fields number to parse (3..12)
+     * Unpacks a basic bed3 [BedEntry] into a bedN+ [ExtendedBedEntry].
```

Comment:
Please update https://github.com/JetBrains-Research/big/blob/master/CHANGES. This fix breaks backward compatibility, so we need to inc major version.

Reply:
I'd argue that it doesn't really break compatibility. Earlier we threw an unspecified exception in an unspecified set of cases. Now we throw a specific exception in specific cases.

Reply:
Before the fix, if a user has fewer fields in the file we will get default values for missing fields so we could open any file as `bed12+`. Now he gets an exception in this case.

Reply:
Not quite true. We couldn't open `chr1 1 100 a b` as `bed12+` because `b` would cause a `NumberFormatException` (which by the way wasn't documented anywhere, if I recall correctly). And this forgiving behaviour itself (filling the default values) wasn't documented either.
Either way, I'm totally fine with a major release, I just want to make a (formal) point that we haven't really broken anything that was previously _promised_.

Reply:
ok